localstack/services/opensearch/cluster.py

```diff
@@ -118,6 +120,66 @@ def build_cluster_run_command(cluster_bin: str, settings: CommandSettings) -> Li
     return [cluster_bin] + cmd_settings
 
 
+def register_cluster(host, path, forward_url, custom_endpoint) -> List[Rule]:
+    from localstack.services.edge import ROUTER
+
+    # custom backends overwrite the usual forward_url
+    forward_url = config.OPENSEARCH_CUSTOM_BACKEND or forward_url
+    endpoint = ProxyHandler(forward_url)
+    rules = []
+    strategy = config.OPENSEARCH_ENDPOINT_STRATEGY
+    # custom endpoints override any endpoint strategy
+    if custom_endpoint:
```

Comment:
Here you also need to check if the custom endpoint is enabled (or you change this param to be of type bool and hand in if the custom endpoint is enabled or not - you aren't using the endpoint URL here anymore anyways).

Reply:
:+1: Thanks, I did confuse those for their endpoint addresses only.